test/core/event_engine/test_suite/event_engine_test_utils.cc

```diff
@@ -108,7 +119,10 @@ absl::Status SendValidatePayload(std::string data, Endpoint* send_endpoint,
   // Wait for async read to complete.
   GPR_ASSERT(read_promise.Get() == true);
   // Check if data written == data read
-  if (data != ExtractSliceBufferIntoString(&read_slice_buf)) {
+  std::string data_read = ExtractSliceBufferIntoString(&read_store_buf);
+  if (data != data_read) {
+    std::cout << "Data written = " << data << std::endl;
```

Comment:
why not use `gpr_log`? If you need to use iostream for some reason, then `std:cerr` should be used for error text.

Reply:
Modified to use gpr_log.